frappe/core/doctype/report/report.json

```diff
@@ -207,7 +207,7 @@
  "idx": 1,
  "index_web_pages_for_search": 1,
  "links": [],
- "modified": "2025-08-28 18:28:32.510719",
```

Comment:
can drop this now since there's no change here

Reply:
Done.